okapi-core/src/main/raml/ModuleUser.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "LaunchDescriptor.json",
+  "title": "LaunchDescriptor",
+  "description": "Specifies how a module is deployed and undeployed",
```

Comment:
Copy/paste error. Does not match file name `ModuleUser.json`.

Reply:
fixed

Reply:
fixed